src/models/invites-ignorer.ts

```diff
@@ -83,6 +94,12 @@ export class IgnoredInvites {
      * @param entity The entity covered by this rule. Globs are supported.
      * @param reason A human-readable reason for introducing this new rule.
      * @return The event id for the new rule.
+     *
+     * # Safety
+     *
+     * This method will rewrite the `Policies` object in the user's account data.
+     * This rewrite is inherently racy and could overwrite or be overwritten by
+     * other concurrent rewrites of the same object.
```

Comment:
shouldn't this be part of the description? At the moment it's part of the `@return`